api/src/main/java/eu/europeana/fulltext/api/service/FTService.java

```diff
@@ -525,7 +495,7 @@ public long deleteAnnoPagesWithSources(List<? extends String> sources) {
         return count;
     }
 
-    public void upsertAnnoPage(List<? extends TranslationAnnoPage> annoPageList)
+    public void upsertAnnoPage(List<? extends AnnoPage> annoPageList)
```

Comment:
I did not know what "upsert" meant and after some searching found an explation in MorphiaUtils.UPERST_OPTS. We may want to explain it also here.